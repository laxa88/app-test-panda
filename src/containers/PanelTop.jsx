import React from "react";

const PanelTop = props => <div>Top panel</div>;

export default PanelTop;
