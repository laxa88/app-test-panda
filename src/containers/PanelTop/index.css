.body {
  border-bottom: 2px solid #ccc;
  height: 60px;
}
