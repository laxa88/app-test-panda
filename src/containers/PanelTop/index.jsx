import React from 'react';

import css from './index.css';
import MenuApp from '../../components/MenuApp';
import MenuUser from '../../components/MenuUser';

const PanelTop = () => (
  <div className={css.body}>
    <MenuApp />
    <MenuUser />
  </div>
);

export default PanelTop;
