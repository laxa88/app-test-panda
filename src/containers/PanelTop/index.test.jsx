import React from 'react';
import renderer from 'react-test-renderer';

import PanelTop from './index';

describe('<PanelTop />', () => {
  it('renders', () => {
    const result = renderer.create(<PanelTop />);
    expect(result.toJSON().props).toEqual({ className: 'body' });
  });
});
