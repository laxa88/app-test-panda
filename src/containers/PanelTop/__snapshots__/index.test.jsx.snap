// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PanelTop /> renders 1`] = `
<div
  className="body"
>
  <div>
    Mocked_FontAwesomeIcon (fas,bars)
    Dummy App
  </div>
  <div>
    Mocked_FontAwesomeIcon (fas,user)
    John | Log out
  </div>
</div>
`;
